### DISCUSSION TITLE
doubt: i`m setting up correctly?

### DISCUSSION THREAD

**Discussion by m1guer** [folke/ts-comments.nvim#26] [Q&A] [answered]
init.lua:
```lua
require("ts-comments")
```

plugins/init.lua
```lua
{
    "folke/ts-comments.nvim",
    opts = {},
    event = "VeryLazy",
    enabled = vim.fn.has "nvim-0.10.0" == 1,
  }
```

and when i comment a tsx or jsx file returns me this:
![image](https://github.com/folke/ts-comments.nvim/assets/91486755/932542f1-1a8d-4549-b588-032bd1752e93)

**Comment by folke** [comment]
Did you install the correct treesitter parsers for tsx and typescript?
What happens when you do `:InspectTree`?
You are on Neovim >= 0.10 right?

**Reply by m1guer** [reply]
You are on Neovim >= 0.10 right? -> yes
Did you install the correct treesitter parsers for tsx and typescript? -> yes

and returns this:
![image](https://github.com/folke/ts-comments.nvim/assets/91486755/0f8b3016-effb-429b-b00a-355e142e9ea5)

**Comment by m1guer** [comment, marked as answer]
hey, the problem is on my side.
just solved unisntalling the old Comment plugin who is installed on my lazy configs.

**Comment by m1guer** [comment]
sorry for the boring
